### PR TITLE
I18n sync In & Up 08/07

### PR DESCRIPTION
**Pull request by deploy-code-org**
# I18n Sync In & Up

This PR contains all changes to internationalization source strings made since the last sync (usually a week).

## To Review:

1. Look through the changes in each of the commits and verify that all changes are expected

   - We expect content changes to come in on a regular basis; individual strings will get updated, added, and deleted.

   - We _do not_ usually expect to see "comprehensive" changes - or changes that apply to large numbers of strings or whole categories of strings all at once - unless we have made specific code changes.

2. Notify the International Partners team about any changes that might appear to manifest as "lost translations"

   - For example: https://github.com/code-dot-org/code-dot-org/pull/31031/files#diff-aa1910a8cf8f9290ae3baf7e39dbae3eL397-R398

     This is a change that added new strings (with both new keys and new content) and simultaneously removed some very similar old ones; we can reasonably infer that the intent here was to swap the new strings out for the old ones. Unfortunately, because the swap also included a content change to the strings the crowdin duplication system won't automatically apply translations to the new strings and so they will need to be manually re-translated.

## To Deploy:

Once one or two people have reviewed and approved this change and the tests are passing, ship it!

It's not necessary to wait for everyone on the team to review.

### CODE REVIEW

File: i18n/locales/source/blockly-mooc/fish.json [hunk 1]
```diff
@@ -1,106 +1,2 @@
 {
-    "fishvtrash-training-init1": "Garbage dumped in the water affects marine life.  In this activity, you will program or train A.I. (artificial intelligence) to identify fish or trash.  Let's clean up the ocean!  Click anywhere on the screen to continue.",
```
Comment (mgc1194) [inline i18n/locales/source/blockly-mooc/fish.json:2]: Flagging these changes as its odd to have all the strings in this file removed.

**Review by mgc1194 — CHANGES_REQUESTED** [review]
We need to check that fish.json has not been deleted accidentally.

**Comment by deploy-code-org** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 53178 -c`

Top 5/8 files sorted by the number of changes:
- `i18n/locales/source/dashboard/scripts.yml`, 307 changes (303 additions, 4 deletions).
- `i18n/locales/source/blockly-mooc/fish.json`, 104 changes (0 additions, 104 deletions).
- `i18n/locales/source/hourofcode/en.yml`, 30 changes (17 additions, 13 deletions).
- `i18n/locales/source/dashboard/courses.yml`, 18 changes (18 additions, 0 deletions).
- `i18n/locales/source/blockly-mooc/aichat.json`, 5 changes (4 additions, 1 deletions).

Top 3/3 commits sorted by the number of changes:
- Commit: dashboard i18n sync
  - 332 changes (327 additions, 5 deletions).
  - 4 files changed.
  - File with the most changes: `i18n/locales/source/dashboard/scripts.yml`, 307 changes (303 additions, 4 deletions).
- Commit: apps i18n sync
  - 114 changes (9 additions, 105 deletions).
  - 3 files changed.
  - File with the most changes: `i18n/locales/source/blockly-mooc/fish.json`, 104 changes (0 additions, 104 deletions).
- Commit: hoc i18n sync
  - 30 changes (17 additions, 13 deletions).
  - 1 files changed.
  - File with the most changes: `i18n/locales/source/hourofcode/en.yml`, 30 changes (17 additions, 13 deletions).

File: i18n/locales/source/blockly-mooc/fish.json (hunk 1 above)
Comment (artem-vavilov) [reply i18n/locales/source/blockly-mooc/fish.json:2]: The fix: https://github.com/code-dot-org/code-dot-org/pull/53191

**Comment by mgc1194** [comment]
Indeed the fish.json file should have not been erased. It got overwritten by the empty file in apps instead of maintaining the content from the external repo.

**Pull request closed without merging**